eppo/src/androidTest/java/cloud/eppo/android/EppoClientTest.java

```diff
@@ -185,6 +195,76 @@ public void testAssignments() {
         runTestCases();
     }
 
+    @Test
+    public void testGracefulModeOn() {
+        try {
+            initClient(HOST, false, true, true);
+            System.out.println("Sleeping for a bit to wait for cache population to complete");
+            Thread.sleep(1000);
```

Comment:
🤨 Unrelated to this PR, but there is probably a better way to do this where we continuously check sleeping 100ms or something much smaller. Sleeps in an automated test are almost always a smell.